From d3c135201889d1e66208126b56cc5317295f3093 Mon Sep 17 00:00:00 2001
From: Daniel Erez <derez@redhat.com>
Date: Wed, 25 Jul 2018 11:24:33 +0300
Subject: [PATCH] v2v: rhv plugin - case-sensitive search queries

Changed both search queries to case-sensitive (to ensure an exact match).
---
 v2v/rhv-upload-plugin.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/v2v/rhv-upload-plugin.py b/v2v/rhv-upload-plugin.py
index 8274f1ee1..4fad27fb8 100644
--- a/v2v/rhv-upload-plugin.py
+++ b/v2v/rhv-upload-plugin.py
@@ -73,7 +73,7 @@ def find_host(connection):
     storage_name = params['output_storage']
     data_centers = system_service.data_centers_service().list(
         search='storage.name=%s' % storage_name,
-        case_sensitive=False,
+        case_sensitive=True,
     )
     if len(data_centers) == 0:
         # The storage domain is not attached to a datacenter
@@ -87,7 +87,7 @@ def find_host(connection):
     hosts_service = system_service.hosts_service()
     hosts = hosts_service.list(
         search="hw_id=%s and datacenter=%s and status=Up" % (vdsm_id, datacenter.name),
-        case_sensitive=False,
+        case_sensitive=True,
     )
     if len(hosts) == 0:
         # Couldn't find a host that's fulfilling the following criteria:
-- 
2.17.2

